apps/next-app/src/components/intro/imagePanel.tsx

```diff
@@ -0,0 +1,9 @@
+import { IntroPanelImage } from "@namviet-fe/core-ui";
```

Comment:
this file splitting is unnecessary here I think. Mình bỏ hình này ngoài page cũng được e ơi, a thấy k cần thiết tách ra vậy lắm